package/Dockerfile.shipyard-dapper-base

```diff
@@ -5,10 +5,10 @@ FROM fedora:35
 ARG UPX_LEVEL=-5
 ENV DAPPER_HOST_ARCH=amd64 SHIPYARD_DIR=/opt/shipyard SHELL=/bin/bash \
     DAPPER_RUN_ARGS="--net=kind"
-ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH} PATH=/go/bin:/usr/local/go/bin:$PATH \
+ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH} PATH=/root/.local/bin:/go/bin:/usr/local/go/bin:$PATH \
```

Comment:
Do we need this? `kind.sh` is executed without relying on `PATH`; is anything else added to `/root/.local/bin`?

Reply:
> kind.sh is executed without relying on PATH 

Is it? When I remove that dir from PATH the kind script cannot use the `j2` binary which is installed by DNF at `/root/.local/bin/j2`  in the dapper build container.